package.json

```diff
@@ -2,5 +2,8 @@
   "devDependencies": {
     "husky": "^5.0.6",
     "prettier-plugin-java": "^1.0.1"
```

Comment:
Prettier recommends installing an [exact version](https://prettier.io/docs/en/install.html#summary) of Prettier incase styling changes between versions.  The difference being the install should be `npm install --save-dev --save-exact prettier-plugin-java`.

The [caret will mean](https://docs.npmjs.com/cli/v6/using-npm/semver#caret-ranges-123-025-004) that some people might potentially get a different version if they installed without a `package-lock.json` file